package.json

```diff
@@ -31,6 +31,7 @@
   },
   "homepage": "https://github.com/cgewecke/eth-gas-reporter#readme",
   "dependencies": {
+    "@codechecks/client": "^0.1.0",
```

Comment:
this should be a peer dependency I think.

Reply:
and possibly dev dep

Reply:
Ah yes! Thanks.